src/app.js

```diff
@@ -275,6 +297,20 @@ function wsConnectionHandler(client, upgradeReq) {
     demuxSink.on('close-sink', ({ id, meta }) => {
         logger.info('[App] Queue for processing id %s', id);
 
+        let isJaaSTenant = false;
```

Comment:
A JaaS meeting url encodes a lot of information, maybe we can extract that knowledge in a JaasUri utilty so that it can be reused elsewhere?

Reply:
you're right, moved to an utility function.